OvmfPkg/ResetVector/ResetVector.nasmb

```diff
@@ -106,6 +106,20 @@
   %define TDX_EXTRA_PAGE_TABLE_BASE FixedPcdGet32 (PcdOvmfSecGhcbPageTableBase)
   %define TDX_EXTRA_PAGE_TABLE_SIZE FixedPcdGet32 (PcdOvmfSecGhcbPageTableSize)
 
+  ;
+  ; TdMailboxBase [0x10, 0x800] is reserved for OS

```

Comment:
These two decriptions (TdMailboxBase [0x10, 0x800]  and TdMailboxBase [0x10,0x20]) conflicts.